logic/account.js

```diff
@@ -306,7 +306,7 @@ class Account {
 			}
 		});
 
-		let limit = constants.ACTIVE_DELEGATES;
+		let limit = ACTIVE_DELEGATES;
```

Comment:
I don't see any usage of limit in this file, can you remove it if its not being used.

Reply:
@ManuGowda it is being used [here](https://github.com/LiskHQ/lisk/blob/2381-prefer-deconstruction/logic/account.js#L344)

Reply:
ok, can change let to const its not being reassigned.

Reply:
@yatki did you address the limit to `const`?

Reply:
@ManuGowda it is being reassigned [here](https://github.com/LiskHQ/lisk/blob/2381-prefer-deconstruction/logic/account.js#L319)